vue.js-case-study.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Vue.js Case Study
+author: Lars Halvor Vikse Kallåk <lhvk>
+tags: javascript, framework, vue
+---
+
+## Introduction
+ 
+ ![Image of Vue logo](https://res.cloudinary.com/practicaldev/image/fetch/s--q6AIFazN--/c_imagga_scale,f_auto,fl_progressive,h_420,q_auto,w_1000/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/ll22z8mea106b4vjdjy7.jpeg)
+
+I've chosen Vue as my case study because frontend is what I study. I've never used Vue and I have zero experience with it, so I reckon this would be pretty interesting to dive deeper into.
```

Comment:
Avoid using the first person, or including opinions in your work. The introduction should explain why Vue is worth learning about for a front end developer. What is Vue? How is it different to it's competitors?